nms/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -126,12 +130,27 @@ const useStyles = makeStyles(theme => ({
   rowId: {
     color: colors.primary.comet,
   },
+  uploadDialog: {
+    width: '800px',
+  },
+  uploadInstructions: {
+    marginTop: '15px',
```

Comment:
By convention, use multiples of 4px
```suggestion
    marginTop: '16px',
```